os/include/tinyara/mm/mm.h

```diff
@@ -783,6 +783,24 @@ void *zalloc_at(int heap_index, size_t size);
  * @endcond
  */
 
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(__KERNEL__)
+
+#define MM_PART_FREE    0
+#define MM_PART_USED    1
+
+struct mm_partition_s {
+        uint32_t start;         /* Start address of the partition */
+        uint32_t size;          /* Size of the partition in bytes */
+        uint8_t status;         /* Current status of the partition, free / used */
+};
+
+/* Functions contained in mm_partition_mgr.c **************************************/
+void mm_initialize_partitions();
+int8_t mm_allocate_partition(uint32_t **start_addr, uint32_t *size);
```

Comment:
Let's include <stdint.h> to use `int8_t`.

Reply:
Ok. done